music21/metadata/__init__.py

```diff
@@ -1273,6 +1312,11 @@ def composers(self):
         '''
         return self._getPluralAttribute('composer')
 
+    @property
+    @deprecated('v8', 'v9', 'use `md.dateCreated` instead')
```

Comment:
ack, unfortunately we can't deprecate properties.  We'll just need to put it in a docstring.
(otherwise people using IDEs to debug will get deprecation errors everywhere, since many IDEs query all properties).  Will give two versions to remove since others might not see it.